configuration/config2/XSDSchemaParser.cpp

```diff
@@ -650,15 +650,29 @@ void XSDSchemaParser::parseAllowedValue(const pt::ptree &allowedValueTree, Schem
     //
     // Parse the value for the enumeration, the add to the allowed values for the limits for this type. Note that enumerations
     // are enhanced with additional information for the UI.
-    allowedValue.m_value = allowedValueTree.get("<xmlattr>.value", "");
+    allowedValue.m_value = allowedValueTree.get("<xmlattr>.value", "missing");
```

Comment:
is it ever possible <xmlattr>.value == "missing"? if there's a chance, let's make the default value much more unique and less likely to be  mis-interpreted. e.g. "hpcc-missing-value", etc